_includes/context/priority.md

```diff
@@ -1,3 +1,97 @@
+<a class="anchor" id="contexttoolbarpriority"></a><a class="anchor" id="contextformpriority"></a>
+
+### Troubleshooting context toolbar and context form conflicts
+
+There are situations where custom context toolbars or custom context forms may conflict with:
+
+  - Context toolbars or context forms provided by the editor,
+  - Context toolbars or context forms provided by the [Quick Toolbars (`quickbars`) plugin]({{site.baseurl}}/plugins/quickbars/),
+  - Other custom context toolbars or custom context forms.
+
+{% if site.productmajorversion < 6 %}
+How these conflicts are resolves depends on the {{site.productname}} version.
+
+- For {{site.productname}} 5.3 or newer, see: [Determining the display priority of context toolbars and context forms]().
+- For {{site.productname}} 5.0, 5.1, or 5.2; see: [Determining the display priority of context toolbars and context forms (Legacy behavior)]().
+{% endif %}
+
+#### Determining the display priority of context toolbars and context forms
+
+There are three settings that determine the priority of context toolbars and context forms: `scope`, `predicate`, and `position`.
+
+- `scope` - Sets the context toolbar or form as visible of certain content (`node`) or a general (or global) toolbar or form (`editor`).
+- `predicate` - Sets a function that tests in the current context position against one or more conditions, returning a boolean value.
```

Comment:
Is the "Sets" necessary? Consistency, yes, but it reads a bit weird to me

Reply:
yeah, removed.